src/client/containers/PullRequestSummary/BuildSection/index.js

```diff
@@ -1,65 +1,59 @@
 /* @flow */
 
-import React from 'react'
-import Col from 'react-bootstrap/lib/Col'
-import Row from 'react-bootstrap/lib/Row'
-import ListGroupItem from 'react-bootstrap/lib/ListGroupItem'
-import pureComponent from 'universal/react-pure-render'
-
-const subHeader = text => (
-  <div style={{ color: '#8c8c8c', fontSize: '13px' }}>
-    {text}
-  </div>
-)
-
-const success = {
-  borderLeft: '5px solid rgb(214, 247, 229)',
-}
-
-const headerColumnStyle = {
-  textTransform: 'uppercase',
-  color: '#10121b',
+import React, { PureComponent } from 'react'
+import { connect } from 'react-redux'
+import { fetchLatestBuildsForRevision } from 'ducks/bfstats/actions'
+import { getPullRequest } from 'ducks/pullrequests/selectors'
+import { createSelector } from 'reselect'
+// import { Builds } from 'containers/Builds'
+
+
+type BuildType = {
```

Comment:
Yes, it's possible that we'll reuse the exact type other components, but before it appears to be likely reused I don't recommend putting it somewhere in common place. This applies to all flow types. 

Why is that?  

For example, we have PullRequest type, used in various UI components, and we might think that we need put it in the common place. But not all components required same fields, perhaps all of them need some particular portion(combination of fields) of that type to render. Having this type generalized will make `flow` validate on all fields and us marking those fields as optional, that what I would like us avoid. Otherwise, it doesn't make sense to use flow at all. But there are indeed places where we could reuse types.